services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java

```diff
@@ -50,7 +50,7 @@
             value = {"/vpcs/{vpcId}/routes/{routeId}"})
     public RouteWebJson getRuleByVpcId(@PathVariable String vpcId, @PathVariable String routeId) throws Exception {
 
-        RouteWebObject routeState = null;
+        Route routeState = null;
```

Comment:
@kevin-zhonghao The change I was referring to was "RouteEntity route = null" :-)

Reply:
Sorry ~ I will change it right now